js/scripts.js

```diff
@@ -0,0 +1,16 @@
+var a = '2',
+    b = '4',
+    value = (a*a) + (2 * a * b) - (b*b)
+;
```

Comment:
unikaj deklaracji 'po przecinku' - wiem, że tak gdzieś tam jest w materiałach ale to śliski temat bardzo. Stosuj:
```
var zmienna1 = 'x';
var zmienna2 = 'y';
var zmienna3 = 'z';
```